tasks.py

```diff
@@ -375,6 +375,14 @@ def test_module(module=None, verbosity=2):
 
 
 @task
+def jshint():
+    """Run the jshint."""
+    for dirpath, dirnames, filenames in os.walk("."):
```

Comment:
This shouldn't be necessary. `jshint` can take a directory, and it will recurse through it.